web/angular-wallet/src/app/store/store.service.ts

```diff
@@ -236,4 +236,47 @@ export class StoreService {
             }
         });
     }
+
+    /*
+    * Method reponsible for fetching block(s) from the database.
+    */
+    public getBlocks(blockHeight: number, count: number=1): Promise<Block[]> {
+        return new Promise((resolve, reject) => {
+            if (this.ready.value) {
+                const blocks = this.store.getCollection('blocks');
```

Comment:
I assume loki does some LRU stuff under the hood, but this will only be an issue if someone leaves the app open for a really, really long time anyway.